server/static/js/main.js

```diff
@@ -19,3 +19,18 @@ if (typeof HTMLElement.prototype.removeClass !== "function") {
     }
 }
 
+$('body').on('click', 'button[data-confirm]', function(e) {
+  e.preventDefault();
+
+  var form = $(this).parents('form');
+  var confirmText = $(this).attr('data-confirm');
+
+  swal({
+    title: confirmText,
```

Comment:
Might be better as the message instead of the title. I'll play around with it live and see which one is better.